db/SDC_postgreSQL/seed.js

```diff
@@ -0,0 +1,56 @@
+const pgp = require('pg-promise')({
+  capSQL: true // generate capitalized SQL 
+});
+const faker = require('faker');
+const { generateOneSimilarListing } = require('./dataGenerator');
+
+const cn = {
+  host: 'localhost',
+  port: 5432,
+  database: 'seabnb',
+  user: 'student',
+  password: 'student',
+};
+
```

Comment:
might look cleaner if you eliminate spacing on variable declarations and keep them at the top